package.json

```diff
@@ -14,6 +14,7 @@
     "@babel/polyfill": "^7.8.7",
     "@babel/preset-env": "^7.9.5",
     "@rdfjs/data-model": "^1.1.1",
+    "@strictsoftware/typedoc-plugin-monorepo": "^0.3.0",
```

Comment:
Changes to this root package.json file should not be needed.

(reminder to check the changes files in a PR every now and then, to make sure that the diff makes sense)